librz/analysis/serialize_analysis.c

```diff
@@ -678,6 +684,74 @@ RZ_API RZ_NULLABLE RzAnalysisVar *rz_serialize_analysis_var_load(RZ_NONNULL RzAn
 	return ret;
 }
 
+enum {
+	VAR_STORAGE_FIELD_TYPE,
+	VAR_STORAGE_FIELD_STACK,
+	VAR_STORAGE_FIELD_REG,
+	VAR_STORAGE_FIELD_OFFSET,
+	VAR_STORAGE_FIELD_EMPTY,
+	VAR_STORAGE_FIELD_COMPOSE,
+	VAR_STORAGE_FIELD_LOCLIST,
+	VAR_STORAGE_FIELD_WAITING,
+};
+
+RZ_API RzSerializeAnalVarParser rz_serialize_analysis_var_storage_parser_new(void) {
```

Comment:
`RzSerializeAnalVarParser` needs to be renamed to `RzSerializeAnalysisVarParser`